db/setupDatabase.js

```diff
@@ -52,6 +52,7 @@ const { DB } = require("../config");
 
   try {
     // Make a temporary client so we can create tables in the database
+    console.log(DB);
```

Comment:
Forgot to turn off logging?

Reply:
Agh, yes. Thank you for catching that. 

This is a good example for static testing. Maybe something like ESLint could pick up mistakes like this.

Reply:
I'm seeing this line of code in the PR, although only in a previous commit. I think the line of code was deleted with the latest commit.